### PR TITLE
Added storage operations

### PR DESCRIPTION
**Pull request by zerc**

### CODE REVIEW

File: pyuploadcare/api_resources.py [hunk 1]
```diff
@@ -711,6 +713,60 @@ class FileList(BaseApiList):
     filters = [('stored', None), ('removed', False)]
 
 
+class FilesStorage(object):
+    """ Object for doing storage-related operations like ``store``, ``delete``
+    for several files.
+    """
+    storage_url = '/files/storage/'
+    uuids_per_storage_request = 100
```
Comment (homm) [inline pyuploadcare/api_resources.py]: Тут все реквесты — строаджи.
`files_per_request` либо `chunk_size`.

File: pyuploadcare/api_resources.py [hunk 2]
```diff
@@ -711,6 +713,60 @@ class FileList(BaseApiList):
...
+
+    def __init__(self, seq):
+        """ Seq can be:
+        * UUIDs
```
Comment (homm) [inline pyuploadcare/api_resources.py]: list of UUIDs

File: pyuploadcare/api_resources.py [hunk 3]
```diff
@@ -711,6 +713,60 @@ class FileList(BaseApiList):
...
+        * File's instances
```
Comment (homm) [inline pyuploadcare/api_resources.py]: list of

File: pyuploadcare/api_resources.py [hunk 4]
```diff
@@ -711,6 +713,60 @@ class FileList(BaseApiList):
     filters = [('stored', None), ('removed', False)]
 
 
+class FilesStorage(object):
+    """ Object for doing storage-related operations like ``store``, ``delete``
```
Comment (homm) [inline pyuploadcare/api_resources.py]: Batch storage operations for the list of files. ``store`` and ``delete`` are supported.

**Comment by homm** [comment]
Looks much better.

**Comment by zerc** [comment]
bump

File: pyuploadcare/ucare_cli.py [hunk 5]
```diff
@@ -58,30 +58,29 @@ def get_file(arg_namespace):
...
-        time_started = time.time()
-        while not file_.is_stored():
-            if time.time() - time_started > timeout:
-                raise TimeoutError('timed out trying to store')
-            file_.update_info()
-            time.sleep(0.1)
 
+def delete_files(arg_namespace):
+    FilesStorage(arg_namespace.paths).delete()
+    _wait_if_needed(arg_namespace, File.is_removed,
```
Comment (homm) [inline pyuploadcare/ucare_cli.py]: Delete is sync. There is no need to wait it.

**Comment by homm** [comment]
But store os not :( (when file is not ready yet)

**Comment by zerc** [comment]
> But store is not :( (when file is not ready yet)

Reverted

**Comment by homm** [comment]
LGTM

**Pull request merged**